<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-2CJPQFA" failures="1" tests="4" name="Test" time="43.195" errors="0" timestamp="2022-11-13T10:11:55 IST">
  <testcase classname="fundModule.VerifyUserCanAddFunds" name="VerifyUserCanLogin" time="21.282">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at fundModule.VerifyUserCanAddFunds.VerifyUserCanLogin(VerifyUserCanAddFunds.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- VerifyUserCanLogin -->
  <testcase classname="fundModule.VerifyUserCanAddFunds" name="VerifyUserCanOpenFundWindow" time="0.0">
    <skipped/>
  </testcase> <!-- VerifyUserCanOpenFundWindow -->
  <testcase classname="fundModule.VerifyUserCanAddFunds" name="VerifyUserCanOpenDepositeFundWindow" time="0.0">
    <skipped/>
  </testcase> <!-- VerifyUserCanOpenDepositeFundWindow -->
  <testcase classname="fundModule.VerifyUserCanAddFunds" name="VerifyUserCanFillDepositeDetailsAdnContinue" time="0.0">
    <skipped/>
  </testcase> <!-- VerifyUserCanFillDepositeDetailsAdnContinue -->
</testsuite> <!-- Test -->
